src/components/Accounts/ChargeFeeFine/ChargeForm.js

```diff
@@ -125,6 +124,16 @@ class ChargeForm extends React.Component {
     change('ownerId', ownerId);
   }
 
+  onClose = () => {
+    const { onClose, form: { reset } } = this.props;
+    this._isMounted = false;
```

Comment:
@ymacedor I don't think we need `_isMounted` here. Could you please remove it?

Reply:
@mkuklis I deleted a line and It works fine, I uploaded the change.